src/core/done.js

```diff
@@ -24,7 +24,7 @@ var done = function() {
 	var msg = ''
 	var groupedByFile = {}
 	var msgGrouped
-	var group = this.config.groupOutputByFile
+	var group = 'groupOutputByFile' in this.config ? this.config.groupOutputByFile : true
```

Comment:
`var group = this.config.groupOutputByFile != null ? this.config.groupOutputByFile : true`